### PR TITLE
Image alt_text argument changes

### PR DESCRIPTION
**Pull request by fedirz**

### CODE REVIEW

File: slackblocks/blocks.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ class ImageBlock(Block):
     def __init__(
         self,
         image_url: str,
-        alt_text: Optional[str] = " ",
```
Comment (nicklambourne) [inline slackblocks/blocks.py:254]: While I get your point, it's not actually an optional argument (i.e. `None` is not a valid value, which is what `Optional` means in the typechecker). If you revert this particular change it should still behave as you intend, without being incorrect from a typing perspective.

**Review by nicklambourne — CHANGES_REQUESTED** [review]

File: slackblocks/blocks.py (hunk 1 above)
Comment (fedirz) [reply slackblocks/blocks.py]: Ok, I've removed that particular change.

File: slackblocks/elements.py [hunk 2]
```diff
@@ -433,9 +433,9 @@ class Image(Element):
 
     def __init__(
         self,
-        alt_text: str,
         image_url: Optional[str] = None,
         slack_file: Optional[SlackFile] = None,
+        alt_text: str = " ",
```
Comment (fedirz) [inline slackblocks/elements.py:438]: @nicklambourne Do you want this type to also be `Optional`?

Comment (nicklambourne) [reply slackblocks/elements.py:438]: No, that should be fine as is. I'm a little worried about changing the order of the arguments, as that technically makes it a breaking change and anyone who wasn't using kwargs will now have a broken build.

**Review by nicklambourne — APPROVED** [review]
LGTM, have moved the args back to their original order so as to not be a breaking change.

**Pull request merged**